pjmedia/src/pjmedia-codec/ffmpeg_vid_codecs.c

```diff
@@ -1415,7 +1416,7 @@ static pj_status_t ffmpeg_codec_open( pjmedia_vid_codec *codec,
     PJ_ASSERT_RETURN(codec && attr, PJ_EINVAL);
     ff = (ffmpeg_private*)codec->codec_data;
 
-    pj_memcpy(&ff->param, attr, sizeof(*attr));
+    ff->param = pjmedia_vid_codec_param_clone(ff->pool, attr);
 
     /* Normalize encoding MTU in codec param */
     if (attr->enc_mtu > PJMEDIA_MAX_VID_PAYLOAD_SIZE)
```

Comment:
old code logic  has a problem:  if mtu changed,  not sync the modify to ff->param.